content/feature_matrix/_index.md

```diff
@@ -24,7 +24,6 @@ SpanData (for exporting)|Yes|Yes|Yes|Yes|Yes|Yes
 Status|Yes|Yes|Yes|Yes|Yes|Yes
 <b>Trace Configs</b>|
 TraceParam Limits (Attributes, Annotations, etc.)|Yes|Yes|<span style="color: #CF7675">No</span>|Yes|<span style="color: #CF7675">No</span>|<span style="color: #CF7675">No</span>
-Rate-limited Sampler|<span style="color: #CF7675">No</span>|<span style="color: #CF7675">No</span>|<span style="color: #CF7675">No</span>|<span style="color: #CF7675">No</span>|Yes<span style="color: #6bb1e0"> (requires PSR-6 cache)</span>|<span style="color: #CF7675">No</span>
```

Comment:
Why did we remove `Rate-Limited Sampler`?

Reply:
This feature won't be part of upcoming GA release, will include/consider later in time.

Reply:
Alright, gotcha, thanks for letting me know.